torch/random.py

```diff
@@ -43,9 +43,6 @@ def manual_seed(seed) -> torch._C.Generator:
     if not torch.mps._is_in_bad_fork():
         torch.mps.manual_seed(seed)
 
-    if hasattr(torch, 'xpu') and not torch.xpu._is_in_bad_fork():
-        torch.xpu.manual_seed_all(seed)
-
```

Comment:
Why does XPU not need to set a manual seed?

Reply:
Here will raise an error since `hasattr(torch, 'xpu')` always `True`. But currently `manual_seed_all` and `_is_in_bad_fork` don't exist in torch.xpu module. This error will be triggered via running UT `test_xpu.py`.
We will add them back in PR about `Generator`.

Reply:
I see. So I wonder if you can add some informative comments here to help reviewers to understand the background.